### DISCUSSION TITLE
Deploying the Azure SaaS Development Kit

### DISCUSSION THREAD

**Discussion by 1iveowl** [Azure/azure-saas#218] [Polls]
Hi, 

We'd love to hear about your experience deploying the Azure SaaS Development Kit.

Please also add your comments below. And if you ran into issues, please feel free to open an issue. 

Thank you

**Comment by anz-yogesh** [comment]
I am getting challenge in deployment. Is there any way we can get help in deployment?

**Comment by obarradas** [comment]
i am deploying in windows with wsl and it keeps erroring on touch: cannot touch '/mnt/c/Users/o.barradas/source/repos/azure-saas-vki/src/Saas.Identity/Saas.IdentityProvider/deployment/log/deploy-2024-05-02--10-02-56.log': No such file or directory

**Reply by leo-schick** [reply]
@obarradas can you try to clone the repository locally on WSL or run it on Windows eg with PoweShell or Git Bash?
I think there might be permission issues popping up when you’re running it on WSL but have the data cloned locally on the C:\ drive. I use WSL as well but have cloned it inside WSL and use Docker Desktop with WSL Integration

**Reply by obarradas** [reply]
thanks @leo-schick this was the problem.

**Comment by bhoopaljanga** [comment]
I am deploying in windows with WSLand keep on getting error 
### Critical error ###
Initilization failed
cp: cannot stat '/asdk/src/Saas.Identity/Saas.IdentityProvider/deployment/config/certificate-policy.json': No such file or directory

I don't see any certificate-policy.json file in config directory
